### PR TITLE
Experiments

### PR DESCRIPTION
**Pull request by Waterpicker**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -29,6 +29,7 @@ sourceSets {
 repositories {
     mavenCentral()
     maven("https://jitpack.io")
+    maven("https://maven.thepokecraftmod.com/releases")
```
Comment (JT122406) [inline build.gradle.kts:32]: Do we ever use this?

File: src/assetLoading/java/gg/generations/pokeutils/PixelAsset.java [hunk 2]
```diff
@@ -31,9 +31,16 @@ public PixelAsset(InputStream is, @Nullable String debugName) {
 
             for (var entry : tarFile.getEntries()) {
                 if (entry.getName().endsWith(".glb")) this.modelName = entry.getName();
+                if (entry.getName().equals("config.json")) {
+
```
Comment (JT122406) [inline src/assetLoading/java/gg/generations/pokeutils/PixelAsset.java:35]: Nothing here?

Comment (Waterpicker) [reply src/assetLoading/java/gg/generations/pokeutils/PixelAsset.java:35]: oh derp

File: build.gradle.kts (hunk 1 above)
Comment (Waterpicker) [reply build.gradle.kts:32]: The creator of the library that does jxl loading is weird so hydos host it himself.

Comment (JT122406) [reply build.gradle.kts:32]: ah gotcha

Comment (Waterpicker) [reply build.gradle.kts:32]: yep

**Review by JT122406 — APPROVED** [review]

**Pull request merged**